chapter06/dags/15_ExternalTaskSensor_non_aligned_schedules.py

```diff
@@ -0,0 +1,23 @@
+"""
+    Listing: 6.6
+"""
+
+import pendulum
+from airflow import DAG
+from airflow.operators.empty import EmptyOperator
+from airflow.sensors.external_task import ExternalTaskSensor
+
+with DAG(dag_id="dag1", schedule="0 16 * * *", start_date=pendulum.today("UTC").add(days=-3)):
```

Comment:
Use section number (15) in dag name so it aligns in the UI close to where it is used.

Reply:
agree